sched/Kconfig

```diff
@@ -356,6 +356,10 @@ config IRQCOUNT
 	bool
 	default n
 
+config ARCH_HAVE_CPU_INDEX
+	bool
```

Comment:
and select ARCH_HAVE_CPU_INDEX on a specific arch/chip which implement up_cpu_index